packages/terra-alert/src/terra-dev-site/doc/example/LongTextExample.scss

```diff
@@ -0,0 +1,6 @@
+:local {
+  .button {
```

Comment:
Remove file

We don't need this file anymore - the inline styles that were there before were old and before when we replaced a generic button with a terra-button actually didn't apply anyway and were incorrect.